src/twisted/web/test/test_distrib.py

```diff
@@ -517,13 +518,11 @@ def cbRendered(ignored):
         return result
 
 
+    @skipIf(not pwd, "pwd module required")
```

Comment:
Maybe this can be replaced by a shared skip condition for Unix... most Unix systems have pwd. 
It is missing only on Win